src/Docker.DotNet/Endpoints/IPluginOperations.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Docker.DotNet.Models;
+using System;
+using System.IO;
+using System.Threading;
+
+namespace Docker.DotNet
+{
+    public interface IPluginOperations
+    {
+        /// <summary>
+        /// List plugins.
+        /// </summary>
+        /// <remarks>
+        /// docker plugin ls
+        ///
+        /// HTTP GET /plugins
+        ///
+        /// 200 - No error.
+        /// 500 - Server error.
+        /// </remarks>
+        Task<IList<PluginListResponse>> ListPluginsAsync(PluginListParameters parameters, CancellationToken cancellationToken = default(CancellationToken));
+        
+        /// <summary>
+        /// Install a plugin.
+        /// </summary>
+        /// <remarks>
+        /// docker plugin pull
+        ///
+        /// HTTP POST /plugins/pull
+        ///
+        /// 204 - No error.
+        /// 500 - Server error.
+        /// </remarks>
+        Task InstallPluginAsync(IList<PluginInstallRequestBodyParameters> body, PluginInstallParameters parameters, IProgress<JSONMessage> progress, CancellationToken cancellationToken = default(CancellationToken));
+
+        /// <summary>
+        /// Inspect a plugin.
+        /// </summary>
+        /// <remarks>
+        /// docker plugin inspect
+        ///
+        /// HTTP GET /plugins/{name}/json
+        ///
+        /// 200 - No error.
+        /// 404 - plugin is not installed.
+        /// 500 - Server error.
+        /// </remarks>
+        //Task<PluginInspectResponse> InspectPluginAsync(string id, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
Same here. Just have it throw `NotImplementedException()` if you don't want to finish it